### ISSUE TITLE
Apply the CrossMark logo widget to your HTML pages

### ISSUE THREAD

**Issue by Melissa37** [elifesciences/elife-crossref-feed#14]
CrossRef supplies a templated HTML/JavaScript code widget which will embed the CrossMark logo and dialog functionality into publisher web pages. The latest version of the widget (v12) is displayed below. This may change based on feedback so do check to ensure you are using the latest version and that it points to our production server:

HTML Widget:

v1.3 (better isolation from publisher css and scripts). This is the version in use as of February 2013. Old versions of the widget will continue to work, but they might be conflicting with your existing css and scripts. Note that the widget has been tested with jQuery versions 1.4.4 through 1.9.1 and jQuery UI versions 1.8.7 through 1.10.2.
v1.4 (better isolation from publisher css. Remove some lint from scripts). This is the version in use as of February 2014. Old versions of the widget will continue to work, but if you are still experiencing css conflicts, this may fix them.

<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.4.4/jquery.min.js" ></script>

<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.8.7/jquery-ui.min.js" ></script>

<script type="text/javascript" src="http://crossmark.crossref.org/javascripts/v1.4/crossmark.min.js"></script>

<a id="open-crossmark" href="#" style="padding: 3px 0 13px 0;"><img style="border: 0; display: none;" id="crossmark-icon" src="http://crossmark.crossref.org/images/crossmark_button.png" /></a>

<div id="crossmark-dialog" style="display: none;" title="">

<iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>
